modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * registry/installing-openstack- .adoc
+// * registry/configuring-registry-operator.adoc
+// * registry/configuring-registry-storage-openstack-user-infrastructure.adoc
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-swift-trust_{context}"]
+= Configuring the Image Registry Operator to trust Swift storage
+
+You must configure the Image Registry Operator to trust {rh-openstack-first} Swift storage.
+
+// to allow the client to pull the image layers from the image registry rather than from links directly from Swift.
+
```

Comment:
Does this procedure have prereqs? It looks like there's nothing to change in the command itself, so I suspect there might be. You might want to add them at some point.